avalon/houdini/pipeline.py

```diff
@@ -201,12 +201,11 @@ def containerise(name,
     return container
 
 
-def parse_container(container, validate=True):
+def parse_container(container):
     """Return the container node's full container data.
 
     Args:
         container (hou.Node): A container node name.
-        validate(bool): turn the validation for the container on or off
 
```

Comment:
Should we keep that argument `validate` and just adding deprecation note in doc string for not breaking old stuff for now (at least before version `6.0.0`) ?

Although `parse_container` is not in host API, but I guess some people might using it in their custom code since it's kind of API snubbed useful function in my opinion.

Having that said, I didn't used that argument so it's okay to just remove it on my end.

Reply:
The `parse_container` is not exposed in the API, however it is somehow exposed in the host API for Nuke where it shouldn't be actually. Which now complicates things... 

I'd refrain from the backwards compatibility for something that should not be a part of the API. And maybe even *asap* remove it from `__init__.py` for the Nuke host. I think the only ones using Nuke currently are @tokejepsen and @mkolar (and some of their clients). They also initialized the Nuke host setup so probably can tell us whether it should or should not be exposed for Nuke (and whether they ended up using it - if not, I'd vote for removing it there too.)

*If* in the end we decide it's not part of the API then I would skip the backwards compatibility.

Reply:
Personally I say get rid of it.